WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/LearnMoreRow.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+struct LearnMoreRow: View {
+    let localizedStringWithHyperlink: String
+    @State private var learnMoreURL: URL? = nil
+
+    var body: some View {
+        AttributedText(learnMoreAttributedString)
+            .accentColor(Color(.textLink))
+            .customOpenURL(binding: $learnMoreURL)
+            .padding(.horizontal, Constants.horizontalPadding)
+            .frame(minHeight: Constants.rowHeight)
+            .safariSheet(url: $learnMoreURL)
+    }
+
+    private var learnMoreAttributedString: NSAttributedString {
+        let learnMoreAttributes: [NSAttributedString.Key: Any] = [
+            .font: StyleManager.subheadlineFont,
+            .foregroundColor: UIColor.textSubtle,
+            .underlineStyle: 0
+        ]
+
+        let learnMoreAttrText = NSMutableAttributedString()
+        learnMoreAttrText.append(localizedStringWithHyperlink.htmlToAttributedString)
+        let range = NSRange(location: 0, length: learnMoreAttrText.length)
+        learnMoreAttrText.addAttributes(learnMoreAttributes, range: range)
+
+        return learnMoreAttrText
+    }
+}
+
+private extension LearnMoreRow {
+    enum Constants {
+        static let horizontalPadding: CGFloat = 16
+        static let rowHeight: CGFloat = 44
+    }
+}
+
+struct LearnMoreRow_Previews: PreviewProvider {
+    static var previews: some View {
+        LearnMoreRow(localizedStringWithHyperlink: "<a href=\"https://pe.usps.com/text/imm/immc5_010.htm\">Learn more</a> about International Tax Number")
```

Comment:
nit: Just as an FYI, you can add the modifier `.previewLayout(.sizeThatFits)` to component previews to show them without the whole device frame/size. (Not a big deal!)

Reply:
To be honest: I can't really use Preview since I'm using a M1 machine with Xcode Rosetta and Preview doesn't work 😔 I'd love to update the previews but can't really see how they look so you'll see me neglecting this more often.

Reply:
Oh, I didn't know that preview doesn't work for you! That's too bad, but don't worry about neglecting it — it's more of a tip for making them look nicer than anything really significant.